docker/healthcheck.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+set -e
+
+# 2 minutes are added to the update frequency threshold to make room for slower starts.
+cutoff_duration=$(($GEOIPUPDATE_FREQUENCY * 62 * 60))
```

Comment:
Agreed. GEOIPUPDATE_FREQUENCY is defined as some duration in `hours`. Thus, `$GEOIPUPDATE_FREQUENCY * 62 * 60` causes an offset of 2 minutes per hour. This results in 

`$GEOIPUPDATE_FREQUENCY * 2` minutes (or 144 minutes when GEOIPUPDATE_FREQUENCY == 72)

instead of the expected 2 minutes stated in the comment.

@faktas2's suggestion of `$GEOIPUPDATE_FREQUENCY * 60 * 60 + 120` does result in the desired 2 minute offset.